cron-jobs/checkForLastLogin.js

```diff
@@ -4,26 +4,31 @@ const emailService = require('~/services/email')
 const emailSubject = require('~/consts/emailSubject')
```

Comment:
Check code styling and formatting accordingly to code convention.

Reply:
What do you mean ? Your comment belongs to line 4 or can you more detailed write pls?

Reply:
This concerns your whole code. Please turn on prettier formatting or run `npm run pre-commit` manually before committing.

Reply:
Fixed